doc/api/process.md

```diff
@@ -2044,6 +2044,42 @@ false
 
 See the [TTY][] documentation for more information.
 
+## process.throw(error)
```

Comment:
I'm not sure that `throw()` is a good name because it's already used by the language.

Reply:
I'd do something like `triggerFatalException`